src/main/resources/static/api.yml

```diff
@@ -1964,6 +1993,55 @@ components:
         - retrieved
         - not_found
         - error
+    PersonNeeds:
+      type: object
+      properties:
+        linkedToRiskOfSeriousHarm:
+          type: array
+          items:
+            $ref: '#/components/schemas/PersonNeed'
+        linkedToReoffending:
+          type: array
+          items:
+            $ref: '#/components/schemas/PersonNeed'
+        notLinkedToHarmOrReoffending:
+          type: array
+          items:
+            $ref: '#/components/schemas/PersonNeed'
+      required:
+        - linkedToRiskOfSeriousHarm
+        - linkedToReoffending
+        - notLinkedToHarmOrReoffending
+    PersonNeed:
+      type: object
+      properties:
+        section:
+          type: string
+        name:
+          type: string
+        overThreshold:
+          type: boolean
+        riskOfHarm:
+          type: boolean
+        riskOfReoffending:
+          type: boolean
+        flaggedAsNeed:
+          type: boolean
+        severity:
+          type: string
+        identifiedAsNeed:
+          type: boolean
+        needScore:
+          type: integer
+      required:
+        - section
+        - name
+        - overThreshold
+        - riskOfHarm
+        - flaggedAsNeed
+        - severity
+        - identifiedAsNeed
+        - needScore
```

Comment:
I think we're missing `riskOfReoffending`?

Reply:
Right you are, have added